birth-registration/fhir-document.jsonc

```diff
@@ -327,6 +327,10 @@
           {
             "url": "http://opencrvs.org/specs/extension/migrant-status",
             "valueString": "FOREIGN_RESIDENT | STUDENT | ASYLUM_SEEKER | RESIDENT | VISITOR | TRANSIENT | REFUGEE | OTHER"
+          },
+          {
+            "url": "http://opencrvs.org/specs/extension/patient-occupation",
+            "valueString": "FOREIGN_RESIDENT | STUDENT | ASYLUM_SEEKER | RESIDENT | VISITOR | TRANSIENT | REFUGEE | OTHER"
```

Comment:
The vlaues looks the same as the migrant-status is this a copy paste mistake? Do we have a full list for occupation?

Reply:
If occupation is a free text maybe just note that in the `valueString`

Reply:
corrected